### ISSUE TITLE
Godot freezes and crashes when I click on LightOccluder2D child node, but the game works and runs just fine.

### ISSUE THREAD

**Issue by Godotah** [godotengine/godot#66082]
### Godot version

3.5

### System information

Windows 10

### Issue description

I saw a video where you can click on "Sprite" and create a "LightOccluder2D sibling" that fits your sprite texture's pixels, and it works just fine, but it creates LightOccluder2D node and every time I click on that node, Godot freezes up and crashes. 

### Steps to reproduce

It just happens when I click on the "Imp"'s "LightOccluder2D" node.

### Minimal reproduction project

[project.zip](https://github.com/godotengine/godot/files/9595565/project.zip)

**Comment by kleonc** [comment]
> ### Minimal reproduction project
> 
> [project.zip](https://github.com/godotengine/godot/files/9595565/project.zip)

The `.zip` archive you've uploaded contains only a single `godot.project` file, meaning it doesn't provide a way to reproduce your issue. Please upload a proper MRP.



> and every time I click on that node, Godot freezes up and crashes.

Freezes or crashes? Or freezes and crashes after some time? Any errors?

**Comment by Godotah** [comment]
Sorry. Like this?
[Scratch.zip](https://github.com/godotengine/godot/files/9602761/Scratch.zip)

And it freezes up whenever I click the LightOccluder2D node. I have to exit out. It did not display any errors.

**Comment by kleonc** [comment]
> Sorry. I'm new to this. Like this? [Scratch.zip](https://github.com/godotengine/godot/files/9602761/Scratch.zip)

Yes, this one I can actually run. I've updated the MRP in the OP with this link.

> And it freezes up whenever I click the LightOccluder2D node. I have to exit out. It did not display any errors.

For me it's "just" laggy (low fps) but doesn't actually freeze/crash. Happens, as you've said, in the editor after selecting _the "Imp"'s "LightOccluder2D" node._ It's editor only issue for you too, right? Seems to not lag when running the main scene from the MRP.

**Comment by Godotah** [comment]
Correct. I was testing it just now and when I clicked the imp's LightOccluder2D node, it did "(not responding)", but it eventually resumed and let me tinker around with the LightOccluder2D node UNTIL I tried to toggle its visibility. I'm still having trouble with that.

**Comment by Godotah** [comment]
> > Sorry. I'm new to this. Like this? [Scratch.zip](https://github.com/godotengine/godot/files/9602761/Scratch.zip)
> 
> Yes, this one I can actually run. I've updated the MRP in the OP with this link. And no worries. 🙂
> 
> > And it freezes up whenever I click the LightOccluder2D node. I have to exit out. It did not display any errors.
> 
> For me it's "just" laggy (low fps) but doesn't actually freeze/crash. Happens, as you've said, in the editor after selecting _the "Imp"'s "LightOccluder2D" node._ It's editor only issue for you too, right? Seems to not lag when running the main scene from the MRP.
Correct. I was testing it just now and when I clicked the imp's LightOccluder2D node, it did "(not responding)", but it eventually resumed and let me tinker around with the LightOccluder2D node UNTIL I tried to toggle its visibility. I'm still having trouble with that.

Any luck?

**Comment by Godotah** [comment]
Update: I've done some more tinkering and I think I've concluded that this freezing, lag, etc happens when I set the LightOccluder2D sibling's settings  to :

Simplification: 0.01
Shrink (pixels): 0
Grow (pixels): 0

(It wouldn't let me set Simplification to 0), but I do this so that the occlusion would be drawn over the sprite's pixels, instead of a weird shape or drawn polygons around it.) which was the whole reason I was trying to do the whole "Create LightOccluder2D sibling" thing.

I would appreciate if anyone could help tell me why this is wrong to do (if it is the cause of the problem). Is it a bug or am I not supposed to be able to do this?
